cas-server-support-ldap/src/main/java/org/jasig/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -152,7 +153,7 @@ protected HandlerResult authenticateUsernamePasswordInternal(final UsernamePassw
         final AuthenticationResponse response;
         try {
             logger.debug("Attempting LDAP authentication for {}", upc);
-            final String password = getPasswordEncoder().encode(upc.getPassword());
+            final String password = Strings.nullToEmpty(getPasswordEncoder().encode(upc.getPassword()));
```

Comment:
Alright, it's definitely something in my deployment causing the credentials to be null. Leave this open for a bit while I address my problem and think about if this is needed upstream.